boards/weact/mini_stm32h743/Kconfig.defconfig

```diff
@@ -0,0 +1,27 @@
+# WeAct Studio MiniSTM32H7 board configuration
+
+# Copyright (c) Charles Dias <charlesdias.cd@outlook.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_MINI_STM32H743
+
+config INPUT
+	default y if LVGL
+
+config LV_COLOR_16_SWAP
+	default y if LVGL
+
+config LV_Z_BITS_PER_PIXEL
+	default 16 if LVGL
```

Comment:
Put those under `if DISPLAY`

Reply:
Done. Also, I added the `LV_COLOR_16_SWAP` and `LV_Z_BITS_PER_PIXEL` under `if LVGL`